pkg/scheduler/service/util.go

```diff
@@ -0,0 +1,70 @@
+package service
+
+import (
+	"bytes"
+	"fmt"
+	"hash/crc64"
+	"sort"
+	"strconv"
+
+	mesos "github.com/mesos/mesos-go/mesosproto"
+)
+
+// compute a hashcode for ExecutorInfo that may be used as a reasonable litmus test
+// with respect to compatibility across HA schedulers. the intent is that an HA scheduler
+// should fail-fast if it doesn't pass this test, rather than generating (potentially many)
+// errors at run-time because a Mesos master decides that the ExecutorInfo generated by a
+// secondary scheduler doesn't match that of the primary scheduler.
+func hashExecutorInfo(info *mesos.ExecutorInfo) uint64 {
```

Comment:
for additional context, should probably include a link to mesos master code that performs executor spec validation